cmd/tempo-vulture/main.go

```diff
@@ -311,7 +311,7 @@ func newJaegerGRPCClient(endpoint string) (*jaeger_grpc.Reporter, error) {
 	}
 
 	// new jaeger grpc exporter
-	conn, err := grpc.Dial(u.Host+":14250", dialOpts...)
```

Comment:
We still require the port for the client call, correct?

Reply:
I'd try to keep it backwards compatible. This is a change that will break vultures everywhere. I'd add a port config param that defaults to `14250`

Reply:
We can do that of course, right now if you pass something like this:
```
--tempo-push-url localhost:14250
```

you will end with this:

```
localhost:14250:14250
```
Even more this line is the source of all that confusion:

https://github.com/grafana/tempo/blob/15ba2e14375b1776d9a78c715fa423d5522c2592/cmd/tempo-vulture/main.go#L66